mythril/laser/ethereum/instructions.py

```diff
@@ -1854,6 +1864,21 @@ def call_(self, global_state: GlobalState) -> List[GlobalState]:
             )
             return [global_state]
 
+        if environment.static:
+            if isinstance(value, int) and value > 0:
```

Comment:
I don't think this is possible at a transaction creation level

Reply:
meaning?

Reply:
Ah nvm, github code folding had me confused